packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -203,7 +203,6 @@ function GlobalStylesStyleBook() {
 
 function GlobalStylesBlockLink() {
```

Comment:
Nitpick (non-blocking): The `GlobalStylesBlockLink` component could be a hook since it renders nothing.

Reply:
We need it to be a component in order to have the needed context from `NavigatorProvider`.